src/library/list.lisp

```diff
@@ -411,3 +411,13 @@
       (concatMap f m))
     (define (>> a b)
       (>>= a (fn (_) b)))))
+
+(cl:defun cl-list-to-coalton (list)
+  (cl:if (cl:null list)
+         Nil
+         (Cons (cl:car list) (cl-list-to-coalton (cl:cdr list)))))
+
+(cl:defun coalton-to-cl-list (list)
+  (cl:loop :for xs := list :then (cl:slot-value xs 'coalton-library::_1)
```

Comment:
sry to dog you on this, but please fixup package prefixes and indentation

Reply:
+1

Reply:
That is how emacs formats it lol.